taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.html

```diff
@@ -1,31 +1,63 @@
-  <p>A <strong>light</strong> or <strong>dark</strong> Section directly above me will create a border between both. <small>An <strong>un-styled</strong> Section never has a border.</small></p>
+  <p>A Section of the same style directly after me will create a border between both. <small>Muted sections should <strong>not</strong> appear consectuively.</small></p>
+</section>
+
+<br />
+
+{# The {{ html }} allows DjangoCMS Snippet to pass in class names #}
+<section class="container  o-section {{modifier_class}} {{ html }}">
+  <p><strong>Container Section (First of a Pair)</strong></p>
+  <p>A <code>.container</code> Section has Container padding and a fake border.<small>Muted sections should <strong>not</strong> appear consectuively.</small></p>
+</section>
+<section class="container  o-section {{modifier_class}} {{ html }}">
+  <p><strong>Container Section (First of a Pair)</strong></p>
+  <p>A <code>.container</code> Section has Container padding and a fake border.<small>Muted sections should <strong>not</strong> appear consectuively.</small></p>
+</section>
+
+<br />
+
+{# The {{ html }} allows DjangoCMS Snippet to pass in class names #}
+<section class="o-section {{modifier_class}} {{ html }}">
+  <p><strong>Section (before Muted Section)</strong></p>
+  <p>A pair of <strong>light</strong> or <strong>dark</strong> Sections may have a <strong>muted</strong> Section between them. <small>Muted sections should <strong>not</strong> appear consectuively.</small></p>
+</section>
+<section class="o-section o-section--style-muted">
+  <p><strong>(Muted) Section</strong></p>
+  <p>A <strong>muted</strong> Section would appear between two <strong>light</strong> or <strong>dark</strong> Sections. <small>It would probably not appear between two <strong>muted</strong> Sections.</small></p>
+</section>
+<section class="o-section {{modifier_class}} {{ html }}">
+  <p><strong>Section (before Muted Section)</strong></p>
+  <p>A pair of <strong>light</strong> or <strong>dark</strong> Sections may have a <strong>muted</strong> Section between them. <small>Muted sections should <strong>not</strong> appear consectuively.</small></p>
 </section>
 
 <br />
 
 {# The {{ html }} allows DjangoCMS Snippet to pass in class names #}
 <section class="o-section {{modifier_class}} {{ html }}">
   <p><strong>Section (with Nested Sections)</strong></p>
-  <p>Sections can be nested within me. <small>A nested Section removes the standard Section spacing.</small></p>
+  <p>Sections can be nested within me. <small>A nested Section removes the standard Section spacing.</small> <small>⚠️ It is not used in any design, so its implementation is incomplete (no borders).</small></p>
   <div class="o-section o-section--style-dark">
     <p><strong>(Nested) Dark Section</strong></p>
     <p>I am nested.</p>
   </div>
+  <div class="o-section o-section--style-muted">
+    <p><strong>(Nested) Muted Section</strong></p>
+    <p>I am nested.</p>
```

Comment:
Design and I can't immediately think of a use case for nested muted sections, but are fine with it being an available style. So, I propose making a note of it. But I don't want to deter use of it, so I propose hiding the note.

```suggestion
    <p>I am nested.<!-- <small>No use case for this is expected, but it is available.<small>--></p>
```